dataflow_test.py

```diff
@@ -111,6 +112,27 @@ def test_longer_pipeline():
     assert result == [ (((n+1)*2)-3)/4 for n in the_source ]
 
 
+def test_fork_implicit_pipes():
+
+    # Arguments can be pipes or tuples
+    # tuples get implicitely converted into pipes
+
+    the_source = list(range(10, 20))
+    add_1      = df.map(lambda x: 1 + x)
+
+    tuple_collector = []; tuple_sink = df.sink(tuple_collector.append)
+    pipe_collector  = [];  pipe_sink = df.sink( pipe_collector.append)
+
+    pipe_as_tuple =        (add_1, tuple_sink)
+    pipe_as_pipe  = df.pipe(add_1,  pipe_sink)
+
+    df.push(source = the_source,
+            pipe   = df.fork(pipe_as_tuple,
+                             pipe_as_pipe ))
```

Comment:
Nice test.

I'm just wondering whether `implicit_pipe` and `explicit_pipe` might be better names than `pipe_as_tuple` and `pipe_as_pipe`. Or maybe `implicit_pipe_as_tuple`.